src/app/backend/apihandler.go

```diff
@@ -55,6 +55,17 @@ func CreateHttpApiHandler(client *client.Client) http.Handler {
 			Writes(NamespacesList{}))
 	wsContainer.Add(namespaceListWs)
 
+	logsWs := new(restful.WebService)
+	logsWs.Path("/api/logs").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+	logsWs.Route(
+		logsWs.POST("").
```

Comment:
You're getting the list of logs using POST method. In RESTful APIs you get a list of objects using GET methods. There's a pull request that tackles the same problem and it follows RESTful specs. Take a look: https://github.com/kubernetes/dashboard/pull/94/files

So, if you change `logsWs.POST("")` to `logsWs.GET("/:namespace/:pod")` you'll have proper RESTful API. The other good thing about such approach is that this would be very similar to what Kubernetes core does. They have namespace and pod name in path param and all the rest in query params: http://kubernetes.io/third_party/swagger-ui/#!/api%2Fv1/readNamespacedPodLog

What do you think?